server/controllers/api/special/claims/index.js

```diff
@@ -0,0 +1,42 @@
+const { handleErrorResponse } = require('../../../utils/errorHandlers.js');
+const db = require('server/models');
+const getClaimData = require('server/utils/getClaimData');
+
+/*
+
+  route to get all claims for special
+
+*/
+
+const channelClaims = async ({ ip, originalUrl, body, params }, res) => {
+  const {
+    name,
+    page,
+  } = params;
+
+  if(name == 'trending') {
```

Comment:
should be triple equals?

Reply:
Yes. I was debugging and forgot to change it back :)